config/config.properties

```diff
@@ -72,6 +72,15 @@ pravegaservice.zkURL=localhost:2181
 # Valid values: Positive integer.
 #pravegaservice.zkSessionTimeoutMs=10000
 
+# Enable security for the ZK client.
+#pravegaservice.secureZK=false
```

Comment:
I'm sure this conversation around security being disabled by default is a bad idea. Please create an issue for us to revisit it.

Reply:
Create an issue to track this: https://github.com/pravega/pravega/issues/2822